lambda/storeSimInformationOnomondo.ts

```diff
@@ -24,16 +33,63 @@ if (apiKey === undefined) {
 	throw new Error(`APIKEY undefined`)
 }
 
+const getHistoryTsFunc = getHistoryTs({
+	getSimDetailsFromCache: getSimDetailsFromCache(db, cacheTableName),
+})
+
+const storeHistoricalDataFunc = storeHistoricalDataInDB({
+	tsw,
+	dbName,
+	tableName,
+})
 export const handler = async (event: SQSEvent): Promise<void> => {
 	console.log(JSON.stringify({ event }))
 	for (const message of event.Records) {
 		try {
-			const iccid = message.body.replace(/"/g, '')
+			const body = JSON.parse(message.body)
+			const iccid = body.iccid
+			const historyTs: string | Date = body.lastTs
+			const storeTimestream: boolean = body.storeTimestream ?? true
```

Comment:
Please document the intention of having this lambda called with storeTimestream=true and storeTimestream=false, which is not obvious.